sdk/python/kfp/cli/experiment.py

```diff
@@ -0,0 +1,89 @@
+import click
+import logging
+from tabulate import tabulate
+
+
+@click.group()
+def experiment():
+    """Manage experiment resources"""
+    pass
+
+
+@experiment.command()
+@click.option(
+    '--description',
+    help="Description of the experiment."
+)
+@click.argument("name")
+@click.pass_context
+def create(ctx, description, name):
+    """Create an experiment"""
+    client = ctx.obj["client"]
+
+    response = client.create_experiment(name, description=description)
+    logging.info("Experiment {} has been submitted\n".format(name))
+    _display_experiment(response)
+
+
+@experiment.command()
+@click.option(
+    '--max-size',
+    default=100,
+    help="Max size of the listed experiments."
+)
+@click.pass_context
+def list(ctx, max_size):
+    """List experiments"""
+    client = ctx.obj['client']
+
+    response = client.list_experiments(
```

Comment:
The client.list_experiments() method used here is actually a wrapper around our backend api method client.experimetns.list_experiment(). I personally prefer to use the backend api method directly, since the wrapper method client.list_experments() in _client.py might not get timely updates/maintenance as backend api method. 

E.g., in your case, could you try if the following works?
client = ctx.obj['client']
client.experiments.list_experiment(page_size=max_size, sort_by='created_at desc')

Thanks!

Reply:
Got it. That makes sense. I'll modify code to use the api methods directly.

Reply:
I changed code in 5b16b1a.